NEWS.md

```diff
@@ -0,0 +1,3 @@
+# shiny.blueprint 0.1.0
```

Comment:
Let's also keep a header for changes which are not yet released. From now on, each PR should edit `NEWS.md`. When publishing to CRAN, we'll just change this header to `0.2.0`.

Could you please add a short summary of changes since 0.1.0? Please use a list format and focus on what's important from end user's perspective. Take a look at [changelog](https://github.com/Appsilon/shiny.react/blob/master/NEWS.md) in `shiny.react` for inspiration.

```suggestion
# shiny.blueprint (development version)

# shiny.blueprint 0.1.0
```